packages/core/src/Stores/client-store.js

```diff
@@ -2933,7 +2933,9 @@ export default class ClientStore extends BaseStore {
     }
 
     get is_account_to_be_closed_by_residence() {
-        return this.account_time_of_closure && this.residence && this.residence === 'sn';
+        return this.account_status?.account_closure?.find(
```

Comment:
There are currently multiple mention of the word "Senegal" hardcoded in the code, you will need to update them as well.

Reply:
Nice catch, will update the content as discussed in the thread